server/example/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	grpc_zap "github.com/grpc-ecosystem/go-grpc-middleware/logging/zap"
+	grpc_recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
+	grpc_ctxtags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
+	"github.com/odpf/salt/common"
+	"github.com/odpf/salt/server"
+	commonv1 "go.buf.build/odpf/gw/odpf/proton/odpf/common/v1"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+var Server = &commonv1.Version{
+	Version: "v1.0.2",
+}
+
+var GRPCMiddlewaresInterceptor = grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(
+	grpc_recovery.UnaryServerInterceptor(),
+	grpc_ctxtags.UnaryServerInterceptor(),
+	grpc_prometheus.UnaryServerInterceptor,
+	grpc_zap.UnaryServerInterceptor(zap.NewExample()),
+))
+
+func main() {
+	grpcPort := 8000
+	httpPort := 8080
+	muxPort := 9000
+	gatewayClientPort := grpcPort
+	go grpcS(grpcPort)
```

Comment:
Let's use the fully qualified name? Like `grpcS` to be called `grpcServer`
Similarly for `httpS` and `muxS`